riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/testing/RiptideClientTest.java

```diff
@@ -0,0 +1,41 @@
+package org.zalando.riptide.spring.testing;
+
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.client.AutoConfigureMockRestServiceServer;
+import org.springframework.boot.test.autoconfigure.web.client.RestClientTest;
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.test.context.TestPropertySource;
+import org.zalando.riptide.spring.RiptideAutoConfiguration;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@RestClientTest
+@AutoConfigureMockRestServiceServer(enabled = false) // see RiptideClientAutoConfiguration
+@TestPropertySource(properties = {
+        "riptide.mocked: true"
+})
+@ImportAutoConfiguration({
+        RiptideMockRestServiceServerAutoConfiguration.class,
+        RiptideAutoConfiguration.class
+})
+public @interface RiptideClientTest {
+
+    /**
+     * Specifies the components to test. May be left blank if components will be manually
+     * imported or created directly.
+     *
+     * @return the components to test
+     */
+    @AliasFor(annotation = RestClientTest.class, attribute = "value")
```

Comment:
Any other attributes that are worth having as an alias?

Reply:
I haven't used any other on `@RestClientTest`, but we may decided to just forward all of it?